runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTracker.java

```diff
@@ -191,6 +194,14 @@ public PatchPrediction predictPatch(FarmingPatch patch)
 		int stages = state.getStages();
 		int tickrate = state.getTickRate() * 60;
 
+		if (config.diseasedCropNotification() && state.getCropState() == CropState.DISEASED)
```

Comment:
This method does not run when a crop changes state. It runs when the ui updates, so this will notify you every 30 seconds when you have a diseased plant an the ui is open

Reply:
Thanks for pointing that out, I agree with your comment that this feature (even if corrected) would be confusing.